src/gluonts/time_feature/lag.py

```diff
@@ -122,6 +128,15 @@ def _make_lags_for_month(multiple, num_cycles=3):
             + _make_lags_for_day(offset.n / (60.0 * 24))
             + _make_lags_for_week(offset.n / (60.0 * 24 * 7))
         )
+    # second
+    elif offset_name == "S":
+        lags = (
+            _make_lags_for_second(offset.n)
+            + _make_lags_for_minute(offset.n / 60.0)
+            + _make_lags_for_hour(offset.n / (60.0 * 60.0))
+            + _make_lags_for_day(offset.n / (60.0 * 60.0 * 24))
+            + _make_lags_for_week(offset.n / (60.0 * 60.0 * 24 * 7))
```

Comment:
I think this is excessive. We already saw issue for `minute` frequency.

Going back 4 weeks blows up the memory requirements.

Reply:
yes you are right! removed!